src/js/fake-translator.js

```diff
@@ -57,7 +57,11 @@ const SOURCE_STRINGS = {
     },
     'selectpicker': {
         'noneselected': 'none selected',
-        'numberselected': '__number__ selected'
+        'numberselected': '__number__ selected',
+        'valuehasspaces': {
+            'minimal': 'select one question has an illegal value __value__ that contains a space',
```

Comment:
Since these messages can also be shown without a widget (a regular select multiple e.g.), let's put the 'valuehasspaces' key under 'alert' instead (so 'alert.valuehasspaces.multiple').

Reply:
Would be nice to place __value__ in between double quotation marks and to start sentence with Capital and end with a point.

Reply:
added double quotation marks